apps/vercel/frontend/src/utils/validateApiPathData/validateApiPathData.ts

```diff
@@ -0,0 +1,4 @@
+import { ApiPath } from '@customTypes/configPage';
+
+export const validateApiPathData = (apiPathData: ApiPath[]) =>
+  Array.isArray(apiPathData) && apiPathData.length > 0;
```

Comment:
this validation is what i would like the most input on. Should we show the input field instead of the select field if the array is empty? See my thought process in the PR description, but I think there are pros and cons here.

Reply:
My thoughts:

* Agreed that showing the input field if paths is empty or we hit an error (either)
* But I think we should have a warning message for each and it should be different:
  * if error: "⚠️ We had trouble fetching routes for this Vercel project."
  * if empty: " ⚠️ It looks like you haven't configured any routes yet."
* both case: display instruction text

Reply:
Okay I like the idea of giving the user instruction 👍 will add a follow-up task here